release-plz.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 # set the path of all the crates to the changelog to the root of the repository
-publish_allow_dirty = ["duckdb-vortex/duckdb"]
+publish_allow_dirty = true
```

Comment:
does this need to be so permissive?

Reply:
This is just to temporarily unblock the flow. The scoping before `["duckdb-vortex/duckdb"]` doesn't actually do anything. Once the actual fix lands in release-plz, this line can fully go away.